File: Katas/C#/8KYU/Simple validation of a username with regex.cs
using System;
using System.Text.RegularExpressions;

public class Kata
{
    public static bool ValidateUsr(string username)
    {
        Regex rgx = new Regex("^[a-z0-9_]{4,16}$");
        return rgx.IsMatch(username);
    }
}

//Other Solutions:

//using System.Text.RegularExpressions;

//public class Kata
//{
//    public static bool ValidateUsr(string username)
//    {
//        return Regex.IsMatch(username, "^[a-z0-9_]{4,16}$");
//    }
//}

//public class Kata
//{
//    public static bool ValidateUsr(string Username) => System.Text.RegularExpressions.Regex.IsMatch(Username, "^[a-z0-9_]{4,16}$");
//}

File: Katas/JavaScript/8KYU/Simple validation of a username with regex.js
function validateUsr(username) {
    var res = /^[/_0-9a-z/s]{4,16}$/g;
    return res.test(username);
}

//Other Solutions:

//function validateUsr(username) {
//    return /^[0-9a-z_]{4,16}$/.test(username)
//}

//const validateUsr = username => /^[a-z\d_]{4,16}$/.test(username);